src/Symfony/Bundle/FrameworkBundle/Resources/config/services.php

```diff
@@ -0,0 +1,225 @@
+use Symfony\Component\HttpKernel\Config\FileLocator;
+use Symfony\Component\HttpKernel\DependencyInjection\ServicesResetter;
+use Symfony\Component\HttpKernel\Event\ControllerArgumentsEvent;
+use Symfony\Component\HttpKernel\Event\ControllerEvent;
+use Symfony\Component\HttpKernel\Event\ExceptionEvent;
+use Symfony\Component\HttpKernel\Event\FinishRequestEvent;
+use Symfony\Component\HttpKernel\Event\RequestEvent;
+use Symfony\Component\HttpKernel\Event\ResponseEvent;
+use Symfony\Component\HttpKernel\Event\TerminateEvent;
+use Symfony\Component\HttpKernel\Event\ViewEvent;
+use Symfony\Component\HttpKernel\EventListener\LocaleAwareListener;
+use Symfony\Component\HttpKernel\HttpKernel;
+use Symfony\Component\HttpKernel\HttpKernelInterface;
+use Symfony\Component\HttpKernel\KernelInterface;
+use Symfony\Component\HttpKernel\UriSigner;
+use Symfony\Component\String\LazyString;
+use Symfony\Component\String\Slugger\AsciiSlugger;
+use Symfony\Component\String\Slugger\SluggerInterface;
+use Symfony\Component\Workflow\Event\AnnounceEvent;
+use Symfony\Component\Workflow\Event\CompletedEvent;
+use Symfony\Component\Workflow\Event\EnteredEvent;
+use Symfony\Component\Workflow\Event\EnterEvent;
+use Symfony\Component\Workflow\Event\GuardEvent;
+use Symfony\Component\Workflow\Event\LeaveEvent;
+use Symfony\Component\Workflow\Event\TransitionEvent;
+use Symfony\Contracts\EventDispatcher\EventDispatcherInterface;
+
+return static function (ContainerConfigurator $container) {
+    // this parameter is used at compile time in RegisterListenersPass
+    $container->parameters()->set('event_dispatcher.event_aliases', [
+        ConsoleCommandEvent::class => 'console.command',
+        ConsoleErrorEvent::class => 'console.error',
+        ConsoleTerminateEvent::class => 'console.terminate',
+        PreSubmitEvent::class => 'form.pre_submit',
+        SubmitEvent::class => 'form.submit',
+        PostSubmitEvent::class => 'form.post_submit',
+        PreSetDataEvent::class => 'form.pre_set_data',
+        PostSetDataEvent::class => 'form.post_set_data',
+        ControllerArgumentsEvent::class => 'kernel.controller_arguments',
+        ControllerEvent::class => 'kernel.controller',
+        ResponseEvent::class => 'kernel.response',
+        FinishRequestEvent::class => 'kernel.finish_request',
+        RequestEvent::class => 'kernel.request',
+        ViewEvent::class => 'kernel.view',
+        ExceptionEvent::class => 'kernel.exception',
+        TerminateEvent::class => 'kernel.terminate',
+        GuardEvent::class => 'workflow.guard',
+        LeaveEvent::class => 'workflow.leave',
+        TransitionEvent::class => 'workflow.transition',
+        EnterEvent::class => 'workflow.enter',
+        EnteredEvent::class => 'workflow.entered',
+        CompletedEvent::class => 'workflow.completed',
+        AnnounceEvent::class => 'workflow.announce',
+    ]);
+
+    $container->services()
+
+        ->set('kernel')
```

Comment:
please keep the order of the service definition the same as in the xml file to make reviews and merging easier.

Reply:
@Tobion restored the ordering in 18d152d4efb0c49a9a057e4807dcad1041583a59